packages/design-picker/src/utils/available-designs.ts

```diff
@@ -4,25 +4,24 @@ import type { MShotsOptions } from '@automattic/onboarding';
 
 type MShotInputOptions = {
 	scrollable?: boolean;
-	highRes?: boolean;
 	isMobile?: boolean;
 };
 
 // Used for both prefetching and loading design screenshots
 export const getMShotOptions = ( {
 	scrollable,
-	highRes,
 	isMobile,
 }: MShotInputOptions = {} ): MShotsOptions => {
 	// Take care changing these values, as the design-picker CSS animations are written for these values (see the *__landscape and *__portrait classes)
+
+	// These settings are arbitrary. Switching from the previous settings (1199x3600) to these settings reduces page weight from 70 MB to 16 MB,
+	// These settings are intended to, at best, be temporary. A long-term solution for this should probably have values determined by A/B tests,
+	// end up serving WEBPs instead of JPEGs, spend fewer bits on parts of images that are not displayed, and possibly display fewer images.
 	return {
 		vpw: isMobile ? MOBILE_VIEWPORT_WIDTH : DEFAULT_VIEWPORT_WIDTH,
-		// 1040 renders well with all the current designs, more details in the links below.
-		// See: #77052
 		vph: scrollable ? 1600 : 1040,
-		// When `w` was 1200 it created a visual glitch on one thumbnail. #57261
-		w: highRes ? 1199 : 600,
-		screen_height: 3600,
+		w: 500,
+		screen_height: 1000,
```

Comment:
I think changing the screen_height is displaying more of the page than before. This probably needs to be at least 1100 to use the same size.

Reply:
Yeah, I wasn't sure what to do about this. On a 1x display, any image over `500` to maybe `600` tall is actually wasted page weight, but the scaling we do means that the image is distorted even at `1000`, which is why, when you toggle between the two options in the video, one shows more of the page than the other (and this is also visible in my comparison, where one of the two versions shows more of the page).

I don't understand what's going on here enough to know why we need this height that's close to double the actual displayed resolution (on a 1x display) for the screenshot to display correctly.

Reply:
I don't fully understand it either tbh (perhaps @Automattic/lego might have some ideas). We definitely don't need the full 3600 though.

Reply:
I think the 3600 is for a "long previews" feature. I saw this when I was looking at the history here to see how we got here (apparently previews used to be `600x600`. It seems like, when you mouseover, the thumbnail is supposed to scroll and I saw some videos of this happening, but I wasn't able to trigger this scrolling behavior myself. I'm not sure what's necessary to enable the behavior, but my browser definitely doesn't need a `3600` tall image if most of it is permanently off screen, and even in the cases where the behavior triggers correctly, I'd be interested in seeing the data about the lift we get from having the feature vs. the drag we get from some users not being able to use the page.

[sorry, I originally left this comment in the wrong spot since I replied by email, and I guess that automatically appends your comment to the main issue even when you're replying to a comment on some code]

Reply:
On looking at the actual images that get served up, requesting `w=1199` and `height=3600` actually gives us an image that's `1199x2697`, at least for one of the images. For the arguments in the new version `w=500`, `height=1000`, we get an image that's `500x375`. So passing a huge `height` doesn't necessarily give us a bunch of pixels which users mostly won't see.